Clarify that link handling only works if Edge is the default browseer (#2650)
MicrosoftDocs/edge-developer: microsoft-edge/progressive-web-apps-chromium/how-to/handle-urls.md

<!-- ====================================================================== -->
## Automatic link handling

> The operating system your PWA is installed on will automatically launch your PWA for any link that refers to content that's within the scope of the PWA when either of the following conditions are met:

Reviewer: ```suggestion
The operating system that your PWA is installed on automatically launches your PWA for any link that refers to content that's within the scope of the PWA, when either of the following conditions are met:
```
tweaks to break up long sentence to help parse

* The PWA was installed from the Microsoft Store on Windows.

> * The PWA was installed by using Microsoft Edge and Microsoft Edge is the default browser on the device.

Reviewer: ```suggestion
* The PWA was installed by using Microsoft Edge, and Microsoft Edge is the default browser on the device.
```
explicit phrase boundaries to break up long sentence to help parse

<!-- ====================================================================== -->
## Automatic link handling

> When Microsoft Edge is your default browser and you install a PWA using Microsoft Edge, all the links that refer to content that's within the scope of this PWA will automatically launch it instead of the web browser.

Reviewer: ```suggestion
When Microsoft Edge is your default browser and you install a PWA using Microsoft Edge, all the links that refer to content that's within the scope of this PWA will automatically launch the PWA instead of launching the web browser.
```
i try to avoid the indirection pointer "it"
